zk-keygen/src/main.rs

```diff
@@ -70,6 +76,72 @@ fn app(crate_version: &str) -> Command {
                 .key_generation_common_args()
                 .arg(no_outfile_arg().conflicts_with_all(&["outfile", "silent"]))
         )
+        .subcommand(
+            Command::new("pubkey")
+                .about("Display the pubkey from a keypair file")
+                .disable_version_flag(true)
+                .arg(
+                    Arg::new("type")
+                        .index(1)
+                        .takes_value(true)
+                        .possible_values(["elgamal"])
+                        .value_name("TYPE")
+                        .required(true)
```

Comment:
```suggestion
                        .possible_values(["elgamal"])
                        .value_name("TYPE")
```

`.required()` and `.takes_value()` are implicit on positional args